src/os/ui/menu/listmenu.js

```diff
@@ -193,6 +215,52 @@ os.ui.menu.list.onSortSelected = function(event) {
 };
 
 
+/**
+ * Handle the "Color Selected" menu event.
+ *
+ * @param {os.ui.menu.MenuEvent} event The menu event.
+ */
+os.ui.menu.list.onColorSelected = function(event) {
+  var context = event.getContext();
+  if (os.instanceOf(context, os.source.Vector.NAME)) {
+    var source = /** @type {!os.source.Vector} */ (context);
+    if (source && source.getSelectedItems()) {
+      var items = source.getSelectedItems();
+      var callback = goog.bind(
+          function(source, items, color) {
```

Comment:
Inline functions will inherit variables from the parent closure, so it isn't necessary to bind the function here. Your callback will already have access to `source` and `items` via the parent.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Closures

When you do need to use bind, prefer the native `bind` call (`function(...) {}.bind(context, args)`) over `goog.bind`.

Reply:
True that.  Updated here and a few other places that I got this from.